sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -292,6 +292,22 @@ void WinHttpTransport::Upload(std::unique_ptr<_detail::HandleManager>& handleMan
   }
 }
 
+static inline std::string GetHeadersAsString(Azure::Core::Http::Request const& request)
```

Comment:
Move to unnamed namespace, remove static.
it would be nice if you removed static from `MergeMaps` above in this file as well.

Static functions are visible to linker from outside of the cpp file. It can be linked with, names can collide. Even if it is inline - inline is a hint to compiler, not an order.
Everything defined in the unnamed namespace is not visible outside of the .cpp file.